file_store/src/file_sink.rs

```diff
@@ -294,7 +294,7 @@ impl FileSink {
 
     pub async fn maybe_roll(&mut self) -> Result {
         if let Some(active_sink) = self.active_sink.as_mut() {
-            if active_sink.time + self.roll_time > Utc::now() {
+            if active_sink.time + self.roll_time < Utc::now() {
```

Comment:
i assume these timers aren't so precise that we'd need this check to be `<=`? i.e. it's not likely to hit directly on `active_sink.time + self.roll_time == Utc::now()` and keep cycling?